pom.xml

```diff
@@ -2,50 +2,47 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.rdfhdt</groupId>
   <artifactId>hdt-mr</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>2.0</version>

```

Comment:
Why the bump to 2.0? The software remains "alpha quality", according to its authors.